src/components/calcite-button/calcite-button.tsx

```diff
@@ -31,9 +30,15 @@ export class CalciteButton {
   //
   //--------------------------------------------------------------------------
 
+  /** optionally specify alignment of button elements. */
+  @Prop({ reflect: true }) alignment?: ButtonAlignment = "center";
+
   /** specify the appearance style of the button, defaults to solid. */
   @Prop({ reflect: true }) appearance: ButtonAppearance = "solid";
 
+  /** The aria-label attribute to apply to the button or hyperlink */
+  @Prop() ariaLabel?: string;
```

Comment:
Can you rename to just 'label'? `aria-label` is already a global property and we don't want it to be set on the host.